README.md

```diff
@@ -1,57 +1,84 @@
+The `durable-functions` npm package allows you to write [Durable Functions](https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-overview) for [Node.js](https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-node). Durable Functions is an extension of [Azure Functions](https://docs.microsoft.com/en-us/azure/azure-functions/functions-overview) that lets you write stateful functions and workflows in a serverless environment. The extension manages state, checkpoints, and restarts for you. Durable Functions' advantages include:
 
-🚧 This library currently in **public preview** status. 🚧
+* Define workflows in code. No JSON schemas or designers are needed.
+* Call other functions synchronously and asynchronously. Output from called functions can be saved to local variables.
+* Automatically checkpoint progress whenever the function schedules async work. Local state is never lost if the process recycles or the VM reboots.
 
-Not all functionality has been implemented yet and there may be significant changes to both this library and the protocol.
+You can find more information at the following links:
+
+* [Azure Functions overview](https://docs.microsoft.com/en-us/azure/azure-functions/functions-overview)
+* [Azure Functions JavaScript developers guide](https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-node)
+* [Durable Functions overview](https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-overview)
+
+A durable function, or _orchestration_, is a solution made up of different types of Azure Functions:
+
+* **Activity:** the functions and tasks being orchestrated by your workflow.
+* **Orchestrator:** a function that describes the way and order actions are executed in code.
+* **Client:** the entry point for creating an instance of a durable orchestration.
+
+Durable Functions' function types and features are documented in-depth [here.](https://docs.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-types-features-overview)
 
 ## Getting Started
 
-1. Install Durable Functions
+You can follow the [Visual Studio Code quickstart](https://docs.microsoft.com/en-us/azure/azure-functions/durable/quickstart-js-vscode) to get started with a function chaining example, or follow the general checklist below:
 
-Run this command from the root folder of your functions app:
-```
-func extensions install -p Microsoft.Azure.WebJobs.Extensions.DurableTask -v 1.6.2
+1. Install prerequisites:
```

Comment:
I feel there is too much redundancy with the content on docs.microsoft.com, which will make it hard to maintain and has the potential to confuse readers. Ideally this repo is a place where _contributors_ can come and learn, but app developers should go either to the official docs or to the extension repo.

So I recommend we reduce the scope of this README as follows:

1. Brief explanation of what this repo is for and how it relates to azure-functions-durable-extension
2. Brief explanation of how JS is special (quick notes on the use of generators, etc.)
3. Show a few code snippets
4. Direct link to the JS quick-start on docs.microsoft.com